docs/tuning_fledge.rst

```diff
@@ -81,6 +81,61 @@ The south services within Fledge each have a set of advanced configuration optio
 
    The *Statistics Collection* setting will not remove any existing statistics, these will remain and remain to be represented in the statistics history. This only impacts new values that are collected. It is recommended that this be set before a service is started for the first time if the desire it to have no statistics values recorded for either assets or the service.
 
+
+  - *Performance Counters* - This option allows for the collection of performance counters that can be used to help tune the south service.
+
+Performance Counters
+--------------------
+
+A number of performance counters can be collected in the south service to help characterise the performance of the service. This is intended to provide input into the tuning of the service and the collection of these counters should not be left on during production use of the service.
```

Comment:
> these counters should not be left on during production 

@MarkRiddoch Do you mean here
a) Its a recommendation/notice to users; Don't use this property in production environment?
May be we can **add as NOTE** here with highlight

May be we can enable/disable this config item on the basis of developer option settings in GUI? That also means we need another optional attribute property required to handle this, as _readonly_ config item is not fit here.